src/basic/missing.h

```diff
@@ -550,6 +550,8 @@ struct btrfs_ioctl_quota_ctl_args {
 #ifndef HAVE_SECURE_GETENV
 #  ifdef HAVE___SECURE_GETENV
 #    define secure_getenv __secure_getenv
+#  elif defined HAVE_ISSETUGID
+#    define secure_getenv(...) (issetugid() ? NULL : getenv(__VA_ARGS__))
```

Comment:
Why not propose a patch that add this one-line function to musl instead? It hardly seems a win to force every secure_getenv user to open-code this.